Blockchain/readme.md

```diff
@@ -109,3 +113,4 @@
   - [Austin Griffith](https://www.youtube.com/channel/UC_HI2i2peo1A-STdG22GFsA)
   - [Nader Dabit](https://www.youtube.com/user/boyindasouth)
   - [Harkirat Singh (Youtube)](https://www.youtube.com/playlist?list=PLVKLWop9wWA82pZoyylZD2VF2c7MR8_5I)
+  - [Code Eater (Hindi)(Youtube)](https://www.youtube.com/@CodeEater21)
```

Comment:
Please remove this resource. We are trying to cater to an international audience, hence we want to keep all resources in English ONLY.

Reply:
Done